src/dotnet/APIView/APIViewUnitTests/APIRevisionsManagerTests.cs

```diff
@@ -43,13 +43,13 @@ public APIRevisionsManagerTests()
 
         // GetLatestAPIRevisionsAsync
 
-        [Fact]
+        /*[Fact(Skip = "Skipping this test because it interface for TelemetryClient")]
```

Comment:
is the idea to eventually enable this again? or do they need a re-write?
could you create a GH tracking issue so we don't forget and add it as part of the reason why it is skipped?

Reply:
https://github.com/Azure/azure-sdk-tools/issues/8873